src/main/kotlin/graphics/scenery/utils/Image.kt

```diff
@@ -10,6 +9,7 @@ import java.awt.image.*
 import java.io.BufferedInputStream
 import java.io.IOException
 import java.io.InputStream
+import java.lang.RuntimeException
```

Comment:
This is weird... does kotlin really make you import `java.lang` packages?

Reply:
It does, but only because this was not using Kotlin's version of it. It was unnecessary and I've removed it.